src/main/java/seedu/phu/model/internship/Phone.java

```diff
@@ -11,8 +11,17 @@ public class Phone {
 
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Phone numbers should only contain numbers, and it should be at least 3 digits long";
-    public static final String VALIDATION_REGEX = "\\d{3,}";
+            "Phone numbers follows the following format: [+COUNTRY_CODE] PHONE_NUMBER\n"
+                    + "They must adhere to the following constraints:\n"
+                    + "1. Country code is optional. "
+                    + "You may include + at the beginning of your phone number to denote the country code.\n"
+                    + "2. The country code can only be 1 to 3 digits long.\n"
+                    + "3. The space in-between the country code and the phone number is optional.\n"
```

Comment:
I think it's better if space is required when user adds country code. Also, need to state that the number of space must be exactly 1.